src/shared/components/search-result/search-result.component.ts

```diff
@@ -146,13 +147,13 @@ export class SearchResultComponent implements OnChanges {
   }
 
   buildCaseField(col: SearchResultViewColumn, result: SearchResultViewItem): CaseField {
-    return {
+    return plainToClass(CaseField, {
```

Comment:
in the methods above we used Object.assign,  can we use the same approach in all the places? Either we use Object.assign everywhere or we use plainToClass

Reply:
done